### ISSUE TITLE
Discord role still assigned to user after bundle is removed from Weezify

### ISSUE THREAD

**Issue by taylorlee3** [riverscuomo/cuomputer#25]
I removed EWBAITE bundle from my Weezify account, but I still have the EWBAITE role on Discord.

**Comment by Jerrykek** [comment]
to add to this issue, recently got the Red-Hurley bundle and did not get the role.

**Comment by DrGLDavidson** [comment]
Messages from Discord related to this issue from Feb 2024:

taylor — 21/02/2024, 00:20
@ Android Do any of you have bundle roles on Discord for bundles you don't own on Weezify?

Jerry — 21/02/2024, 00:21
nope but I am missing roles for bundles I do own.

taylor — 21/02/2024, 00:23
@ Jerry @ Lina lazybones did either of you purchase those bundles after Feb 5th?

taylor — 21/02/2024, 00:26
If you guys have github accounts, can you please add a comment to this issue: https://github.com/riverscuomo/cuomputer/issues/25 .

King Tom — 21/02/2024, 00:24
My bundle-related roles are correct.

Jerry— 21/02/2024, 00:24
yeah, I did.

Jerry— 21/02/2024, 01:53
added a comment.
oh, it seems the issue is resolved. the role was added.

**New related issue:**
Noticed user Jerry (swouscity) has SZNZ role. They do not have it in Weezify.

**Comment by KCreek1** [comment]
I believe Jerry got that role by clicking on the SZNZ bundle in market when the placeholder for the upcoming bundles were briefly showing.

**Comment by KCreek1** [comment]
Yeah. That's what seemed to happen. This is part of the discussion from that time in Black Album Demo Bundle. 

![image](https://github.com/user-attachments/assets/c5fdf080-2fab-4b19-929d-1c192b59ad2e)

![image](https://github.com/user-attachments/assets/6e288577-151e-454a-bb92-cdffe4e88fa6)

**Comment by Jerrykek** [comment]
Yes, I did get the SZNZ role when they were in the shop at some point. I had heard some rumblings of someone actually buying the Van Weezer bundle and didn't get anything but the role, I wanted to see if that was actually true, and it was. Hope this clears anything up.
